server/router/api.go

```diff
@@ -46,13 +46,15 @@ func apiRoutes(e *gin.Engine) {
 			users.DELETE("/:login", api.DeleteUser)
 		}
 
-		orgBase := apiBase.Group("/orgs/:owner")
+		apiBase.GET("/orgs/lookup/*org_full_name", api.LookupOrg) // TODO: check if this public route is a security issue
```

Comment:
if LookupOrg does call the gitea api with no user auth ... yes it is an issue, lookup from DB should be fine

Reply:
That would be the best as orgs could be private and this endpoint would allow enumeration otherwise.